Dockerfile

```diff
@@ -145,9 +145,27 @@ RUN mkdir -p /var/www/cms/library/temp &&  \
     mkdir -p /run/apache2 && \
     rm /etc/apache2/conf.d/info.conf && \
     rm /etc/apache2/conf.d/userdir.conf && \
-    touch /CMS-FLAG && \
-    addgroup ssmtp && \
-    chmod 777 /tmp
+    addgroup ssmtp
+
+# Initial configuration
+RUN test ! -f /var/www/SETUP-COMPLETED && \
```

Comment:
this file won't ever be there when the container builds, so this will always run - can remove the test and file completely?

Reply:
It's created some lines below. but probably a better solution would be to check for the config files itself and copy it if it does not exist.
I guess this was done so in the `entrypoint.sh` like this to not override existing developer configurations or so. If this is not needed, best solution would be to just copy the files without checking anything 😄